units/Powerslave.cfg

```diff
@@ -0,0 +1,13 @@
+#textdomain wesnoth-loti
+[unit_type]
+    id=Powerslave
```

Comment:
I see you've figured out my naming patterns.

Reply:
I haven't noticed any other units named for old Iron Maiden songs (though Prophet_of_Disaster is a line from Die With Your Boots On, not a song title).  But then I gave up on them back in the early 90s when they, well I don't know what they did but it really sucked.

Reply:
I never quoted Iron Maiden specifically, but many names were inspired by metal songs (typically more obscure than Iron Maiden). Iron Maiden fits the scheme.